pkgs/applications/networking/mailreaders/aerc/default.nix

```diff
@@ -61,6 +62,12 @@ buildGoModule rec {
       "${lib.makeBinPath [ ncurses ]}"
     wrapProgram $out/share/aerc/filters/html --prefix PATH ":" \
       ${lib.makeBinPath [ w3m dante ]}
+    wrapProgram $out/share/aerc/filters/html-unsafe --prefix PATH ":" \
+      ${lib.makeBinPath [ w3m dante ]}
+    substituteAllInPlace $out/share/aerc/filters/plaintext --replace "/usr/bin/awk" "${gawk}/bin/awk"
```

Comment:
using `patchShebangs $out/share/aerc` with gawk in buildInputs should fix these automatically `